File: app/src/main/java/com/example/unsplash/model/models/LikePhotoResult.java
package com.example.unsplash.model.models;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class LikePhotoResult {
    @SerializedName("photo")
    @Expose
    public Photo photo;
    @SerializedName("user")
    @Expose
    public User user;
}

File: app/src/main/java/com/example/unsplash/model/models/LikePhotoResult.kt
package com.example.unsplash.model.models

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class LikePhotoResult {
    @SerializedName("photo")
    @Expose
    var photo: Photo? = null
    @SerializedName("user")
    @Expose
    var user: User? = null
}
